packages/dom-helper/lib/main.js

```diff
@@ -242,6 +242,14 @@ if (canRemoveSvgViewBoxAttribute){
 }
 
 prototype.setPropertyStrict = function(element, name, value) {
+  if (value === undefined) {
+    value = null;
+  }
+
+  if (value === null && (name === 'value' || name === 'type' || name === 'src')) {
```

Comment:
Most of this is probably element-type specific, yeah. But I'm also willing to punt on that for now :-p